### PR TITLE
Feature/platform 24 demos deploy staging develop

### PR DESCRIPTION
**Pull request by jrschumacher**
@benlstanley added the deployment to develop and staging

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -89,32 +89,74 @@ steps:
...
+      - *agent_queue
+  - wait
+
+  - label: ':s3: Invalidate CloudFront Cache Staging'
+    command: 'buildkite-scripts/scripts/developer-demos/invalidate-cloudfront-cache.sh'
+    branches: 'develop'
+    env:
+      S3_PATH: 'protect-staging'
+    agents:
+      - *agent_queue
```
Comment (NathanAB) [inline .buildkite/pipeline.yml]: Looks like the two steps above are duplicated

Comment (jrschumacher) [reply .buildkite/pipeline.yml]: @NathanAB Ah good catch!

**Comment by jrschumacher** [comment]
@NathanAB would you check this again?

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -89,47 +86,95 @@ steps:
...
   - wait
 
+  ####
+  # UNBLOCKED DEPLOYMENTS
+  ####
+
+  ####
+  # STAGING
+  # Deploy and Clear Cache
+  ####
```
Comment (NathanAB) [inline .buildkite/pipeline.yml]: Nit but can the deploy and build steps appear in the same order with regard to the environment? Makes it easier to read, i.e. prod, staging, develop or the reverse

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -89,47 +86,95 @@ steps:
...
+    command: 'buildkite-scripts/scripts/developer-demos/deploy-to-s3.sh'
+    branches: 'develop'
+    env:
+      S3_PATH: 'protect-staging'
+    agents:
+      - *agent_queue
+
+  - wait
+
+  - label: ':s3: Invalidate CloudFront Cache Staging'
```
Comment (NathanAB) [inline .buildkite/pipeline.yml]: for consistency
```suggestion
  - label: ':s3: Invalidate CloudFront Cache (STAGING)'
```

File: .buildkite/pipeline.yml [hunk 4]
```diff
@@ -89,47 +86,95 @@ steps:
...
 
+  ####
+  # UNBLOCKED DEPLOYMENTS
+  ####
+
+  ####
+  # STAGING
+  # Deploy and Clear Cache
+  ####
+  - label: ':s3: Deploy to S3 Staging'
```
Comment (NathanAB) [inline .buildkite/pipeline.yml]:
```suggestion
  - label: ':s3: Deploy to S3 (STAGING)'
```

**Review by NathanAB — APPROVED** [review]

**Pull request merged**